### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by emily-watt-visier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,16 +88,16 @@ export VISIER_APIKEY=the-api-key-issued-by-visier
...
 ```sh
 $ source .env
 ```
 
 ## Jupyter Notebooks
 Jupyter notebooks and lab are well-suited to run Visier connector code. However, some users may not find OS-level variables ideal. As of version `0.9.9`, the Visier Python connector supports [dotenv](https://pypi.org/project/python-dotenv/) to facilitate a more dynamic switching of Visier authentication parameters. If the file is called `.env`, the Python package `dotenv` attempts to load the file. If the file has a different name, you must provide that file name when loading the environment with `dotenv`.
 
-### Jupyter Basic Authentication Example
-Basic Authentication is the most practical means of authenticating against Visier for Jupyter notebooks.
+### Jupyter Password Authentication Example
```
Comment (hareskoug) [inline README.md:99]: I don't understand this. Basic Authentication is the official term for username/password auth. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes

File: README.md [hunk 2]
```diff
@@ -62,20 +62,20 @@ with VisierSession(auth) as s:
...
 
-### Basic Authentication
-Though the Visier Python Connector doesn't directly interact with the environment variables, the following list and example below illustrate the basic authentication parameters. These are also the environment variables the `make_auth()` utility function will use.
-* `VISIER_HOST`: The fully qualified domain name and protocol to access your Visier tenant
-* `VISIER_USERNAME`: The user name with sufficient API capabilities
-* `VISIER_PASSWORD`: The password of that user
-* `VISIER_APIKEY`: The API key granted by Visier
-* `VISIER_VANITY`: The readable name of the customer organization
-* `VISIER_TARGET_TENANT_ID`: The technical name of the tenant for the customer. This is only applicable in partner configurations
+### Password Authentication
```
Comment (hareskoug) [inline README.md:67]: Same. Leave all `Basic Authentication` in

**Review by hareskoug — COMMENTED** [review]
Looks good except for the change from Basic -> Password

File: README.md (hunk 1 above)
Comment (emily-watt-visier) [reply README.md:99]: In our api-samples repo, we refer to authorization_code and password auth, which I assumed are the exact same as these 2 options. I changed it here to be more consistent with the other repo: https://github.com/visier/api-samples/tree/main/authentication

Comment (hareskoug) [reply README.md:99]: There's a difference between OAuth grant types (authorization code versus password) and Basic authentication in HTTP.

What do you think about this:
* For anything related to OAuth we're careful to refer to grant types (as opposed to authentication type)
* For legacy ASID authentication, we call it Basic in the sense it's only requiring username and password (even though it does not use the HTTP Basic authorization header. Sigh...)

Comment (emily-watt-visier) [reply README.md:99]: I see! Yup, that distinction sounds good to me.

**Review by hareskoug — APPROVED** [review]

**Pull request merged**